src/kernels/jupyter/launcher/liveshare/hostJupyterExecution.ts

```diff
@@ -24,43 +22,63 @@ import {
     INotebookServerFactory,
     IJupyterServerUriStorage
 } from '../../types';
+import * as urlPath from '../../../../platform/vscode-path/resources';
 import { IJupyterSubCommandExecutionService } from '../../types.node';
 import { JupyterConnection } from '../../connection/jupyterConnection';
+import { PythonEnvironment } from '../../../../platform/pythonEnvironments/info';
+import { DataScience } from '../../../../platform/common/utils/localize';
+import { Cancellation } from '../../../../platform/common/cancellation';
+import { IJupyterConnection } from '../../../types';
+import { JupyterSelfCertsError } from '../../../../platform/errors/jupyterSelfCertsError';
+import { JupyterSelfCertsExpiredError } from '../../../../platform/errors/jupyterSelfCertsExpiredError';
+import { LocalJupyterServerConnectionError } from '../../../../platform/errors/localJupyterServerConnectionError';
+import { RemoteJupyterServerConnectionError } from '../../../../platform/errors/remoteJupyterServerConnectionError';
+import { sendTelemetryEvent, Telemetry } from '../../../../telemetry';
+import { JupyterWaitForIdleError } from '../../../errors/jupyterWaitForIdleError';
+import { expandWorkingDir } from '../../jupyterUtils';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
+const LocalHosts = ['localhost', '127.0.0.1', '::1'];
 
 /**
  * Jupyter server implementation that uses the JupyterExecutionBase class to launch Jupyter.
  */
 @injectable()
-export class HostJupyterExecution extends JupyterExecutionBase implements IJupyterExecution {
+export class HostJupyterExecution implements IJupyterExecution {
```

Comment:
Merged base class code into this class, base class was not used anywhere else, hence no point of the base class